evidence.go

```diff
@@ -109,15 +122,15 @@ func (e *Evidence) Sign(signer cose.Signer) ([]byte, error) {
 
 }
 
-// SignUnvalidated returns the Evidence wrapped in a CWT according to the
+// Sign returns the Evidence wrapped in a CWT according to the
```

Comment:
Noticed my comment about modifying the Interface Interpretation of `ccatoken` is applicable to this interface also,

Also the comment in the Sign interface is mis-leading, as well!